test/DurableTask.SqlServer.AzureFunctions.Tests/TargetBasedScalingTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DurableTask.SqlServer.AzureFunctions.Tests
+{
+    using System;
+    using DurableTask.Core;
+    using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+    using Microsoft.Azure.WebJobs.Host.Scale;
+    using Microsoft.Extensions.Logging;
+    using Moq;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class TargetBasedScalingTests
+    {
+        readonly Mock<SqlMetricsProvider> metricsProviderMock;
+        readonly Mock<IOrchestrationService> orchestrationServiceMock;
+
+        public TargetBasedScalingTests()
+        {
+            this.orchestrationServiceMock = new Mock<IOrchestrationService>(MockBehavior.Strict);
+
+            this.metricsProviderMock = new Mock<SqlMetricsProvider>(
+                MockBehavior.Strict,
+                null,
+                null);
```

Comment:
non-blocking heads up: but I'm a fan of using named parameters whenever `null` is used in a method call so that a reader can know what parameters are being set. Otherwise, it's a bit difficult to examine.

But for this review, don't worry about it.

Reply:
Same! Fixed'ish. One awkward thing in this case is that this is a `params` parameter, which means I can't use named parameters in the way that you would with a normal method call.